### ISSUE TITLE
K11

### ISSUE THREAD

**Issue by Kotsin** [EchoControl/Gunkolle#13]
Can K11 be added for 0-2 dragging?

**Comment by Kotsin** [comment]
Well, I've made it myself by changing an image of FAL with K11, but it still would be great

**Comment by EchoControl** [comment]
Yeah, I don't play gf anymore, so the script will stay in the state it is.

But if you want to add K11 or other 5* or 4* tdolls you can do it pretty easy. Go to `Gunkolle.ahk` and in the `Initialize()` segment you can find a list of tdolls in use:
`5Star = TYPE97,OTS14,HK416,G41,TYPE95,G11,FAL,WA2000
	4Star = `
All caps and name in `5Star` and `4Star` should be the same as it is in the `/pics/DollList` directory.
